contracts/feature-tests/composability/forwarder-interactor/src/interact.rs

```diff
@@ -0,0 +1,2115 @@
+                token_nonce,
+                token_amount,
+            ))
+            .returns(ReturnsResultUnmanaged)
+            .run()
+            .await;
+
+        println!("Result: {response:?}");
+    }
+
+    pub async fn transf_exec_multi_accept_funds(&mut self) {
+        let to = bech32::decode("");
+        let token_payments = MultiValueVec::from(vec![MultiValue3::<
+            TokenIdentifier<StaticApi>,
+            u64,
+            BigUint<StaticApi>,
+        >::from((
+            TokenIdentifier::from_esdt_bytes(&b""[..]),
+            0u64,
+            BigUint::<StaticApi>::from(0u128),
+        ))]);
+
+        let response = self
+            .interactor
+            .tx()
+            .from(&self.wallet_address)
+            .to(self.state.current_address())
+            .gas(80_000_000u64)
+            .typed(proxy::ForwarderProxy)
+            .transf_exec_multi_accept_funds(to, token_payments)
+            .returns(ReturnsResultUnmanaged)
+            .run()
+            .await;
+
+        println!("Result: {response:?}");
+    }
+
+    pub async fn forward_transf_exec_reject_funds_multi_transfer(&mut self) {
+        let to = bech32::decode("");
+        let token_payments = MultiValueVec::from(vec![MultiValue3::<
+            TokenIdentifier<StaticApi>,
+            u64,
+            BigUint<StaticApi>,
+        >::from((
+            TokenIdentifier::from_esdt_bytes(&b""[..]),
+            0u64,
+            BigUint::<StaticApi>::from(0u128),
+        ))]);
+
+        let response = self
+            .interactor
+            .tx()
+            .from(&self.wallet_address)
+            .to(self.state.current_address())
+            .gas(80_000_000u64)
+            .typed(proxy::ForwarderProxy)
+            .forward_transf_exec_reject_funds_multi_transfer(to, token_payments)
+            .returns(ReturnsResultUnmanaged)
+            .run()
+            .await;
+
+        println!("Result: {response:?}");
+    }
+
+    pub async fn transf_exec_multi_reject_funds(&mut self) {
+        let to = bech32::decode("");
+        let token_payments = MultiValueVec::from(vec![MultiValue3::<
+            TokenIdentifier<StaticApi>,
+            u64,
+            BigUint<StaticApi>,
+        >::from((
+            TokenIdentifier::from_esdt_bytes(&b""[..]),
+            0u64,
+            BigUint::<StaticApi>::from(0u128),
+        ))]);
+
+        let response = self
+            .interactor
+            .tx()
+            .from(&self.wallet_address)
+            .to(self.state.current_address())
+            .gas(80_000_000u64)
+            .typed(proxy::ForwarderProxy)
+            .transf_exec_multi_reject_funds(to, token_payments)
+            .returns(ReturnsResultUnmanaged)
+            .run()
+            .await;
+
+        println!("Result: {response:?}");
+    }
+
+    pub async fn change_owner(&mut self) {
+        let child_sc_address = bech32::decode("");
```

Comment:
Is this the empty address, or a zero array?

Reply:
Indeed this will probably throw an error, I will change it manually to Address::zero()